patches/api/0429-Bell-API.patch

```diff
@@ -0,0 +1,67 @@
+From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
+From: Lama06 <andreasprues36@gmail.com>
+Date: Sun, 22 Jan 2023 11:22:49 +0100
+Subject: [PATCH] Bell API
+
+
+diff --git a/src/main/java/org/bukkit/block/Bell.java b/src/main/java/org/bukkit/block/Bell.java
+index 6fee356a3fc2811d99d9ed2a551118e3e6f0de79..3aa5e4ed08e24cdbe90602a4d6f947a7ea45e6a7 100644
+--- a/src/main/java/org/bukkit/block/Bell.java
++++ b/src/main/java/org/bukkit/block/Bell.java
+@@ -3,4 +3,55 @@ package org.bukkit.block;
+ /**
+  * Represents a captured state of Bell.
+  */
+-public interface Bell extends TileState { }
++public interface Bell extends TileState {
++    // Paper start - Bell API
++    /**
++     * Rings the bell in the direction that it is facing.
++     */
++    default void ring() {
++        ring(((org.bukkit.block.data.type.Bell) getBlockData()).getFacing());
++    }
```

Comment:
I don't think this should be a default method, it just is an unneeded conversion to API types when you can just do it in the impl without any API types created.

Reply:
Thanks for all your feedback! I have done everything you suggested now.